app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Welcome#index</h1>
```

Comment:
@jw9billions I'd like to see you fill out this homepage so it looks really welcoming to a user.